### PR TITLE
ARROW-17684: [CI][deb] Disable Flight for arm64

### PR DESCRIPTION
**Pull request by kou**
Because we don't have enough resource to build Flight on Travis CI.

### CODE REVIEW

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

File: dev/tasks/linux-packages/travis.linux.arm64.yml [hunk 1]
```diff
@@ -118,8 +118,8 @@ script:
         --trace \
         {{ task_namespace }}:build \
         BUILD_DIR=build \
-        DEB_BUILD_OPTIONS=parallel=2 \
-        RPM_BUILD_NCPUS=2
+        DEB_BUILD_OPTIONS=parallel=3 \
+        RPM_BUILD_NCPUS=3
```
[the remark this answers is not in the document]
Comment (kou) [reply dev/tasks/linux-packages/travis.linux.arm64.yml:122]: I'm just trying. :-)

I tried `1` 8c1ce5e45817db0a9a043ddaf31ab9cfc2cf67d0 but it also timed out: https://github.com/apache/arrow/pull/14300#issuecomment-1266278233

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

File: dev/tasks/linux-packages/travis.linux.arm64.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pitrou) [reply dev/tasks/linux-packages/travis.linux.arm64.yml]: Then there's probably not enough available RAM on the machine to build some C++ files.

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux-arm64

**Comment by kou** [comment]
@github-actions crossbow submit -g linux

**Comment by kou** [comment]
@github-actions crossbow submit debian-* ubuntu-*

**Comment by kou** [comment]
@github-actions crossbow submit debian-* ubuntu-*

**Comment by kou** [comment]
@github-actions crossbow submit debian-* ubuntu-*

**Comment by kou** [comment]
@github-actions crossbow submit debian-* ubuntu-*

**Comment by pitrou** [comment]
Are there other CI services that we can use for Arm64? It looks like Travis-CI is less and less of an option.

Also cc @assignUser for ideas.

**Comment by pitrou** [comment]
If we are ok with self-hosted runners then I'm sure there are many (paying) options :-)

**Comment by pitrou** [comment]
Oh, I hadn't read carefully: "Oracle Cloud Free Tier services". Interesting.

**Comment by assignUser** [comment]
Yes we have already looked into alternatives/moving away from travis but postponed it for now as there was no immediate need to do it. If this has changed than I can pick it back up. Would of course require converting the travis builds to gha workflows, so not a drop-in replacement.

**Review by kou — COMMENTED** [review]
Anyway, I merge this because Travis CI failures are just upload failures not "no output" error now.

**Pull request merged**